### PR TITLE
Change images to buttons

### LINKED ISSUE #84: Change rps images on play page to actual buttons or add a visual indicator
Buttons have feedback when pressed, images do not

### PR DESCRIPTION
**Pull request by HamdiaA**
Closes #84.

### CODE REVIEW

File: flask/app/templates/base.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <link rel="stylesheet" href="./static/styles/index.css" />
+    <link rel="stylesheet" href="./static/styles/play.css" />
```
Comment (calvh) [inline flask/app/templates/base.html:8]: Please remove this line

Any css defined in `base.html` will be loaded by every page that is based on it. `play.css` should only be loaded for the Play page, not for every page.

**Review by calvh — CHANGES_REQUESTED** [review]
Thanks @HamdiaA 

Please fix some minor issues

File: flask/app/templates/base.html (hunk 1 above)
Comment (HamdiaA) [reply flask/app/templates/base.html]: done 👍🏼

**Review by Rebecca-CYLi — APPROVED** [review]

**Review by calvh — APPROVED** [review]

**Comment by calvh** [comment]
thanks @HamdiaA

**Pull request merged**